### PR TITLE
ENYO-2547 Fix Slider to work with popup: false

### PR DESCRIPTION
**Pull request by webOS101**
Enyo-DCO-1.1-Signed-off-by: Roy Sutton <roy.sutton@lge.com>

### CODE REVIEW

File: src/Slider/Slider.js [hunk 1]
```diff
@@ -493,17 +493,37 @@ module.exports = kind(
...
 
+	/**
+	* @private
+	*/
+	updateValues: function (value) {
 		if (this.lockBar) {
-			this.setProgress(this.getValue());
+			this.setProgress(value);
+		} else {
+			if(this.popup) {
```
Comment (Djspaceg) [inline src/Slider/Slider.js]: Spacing update: `if (` ;)

File: src/Slider/Slider.js [hunk 2]
```diff
@@ -581,13 +598,20 @@ module.exports = kind(
...
-	updatePopupPosition: function (percent) {
+	updateKnobPosition: function (percent) {
 		this.$.knob.applyStyle(this.get('orientation') == 'vertical' ? 'bottom' : 'left', percent + '%');
 	},
 
 	/**
 	* @private
 	*/
+	updatePopupPosition: function () {
+		;	// Do not allow ProgressBar to update popup position
```
Comment (Djspaceg) [inline src/Slider/Slider.js]: I don't think empty methods is a practice that we endorse. Can we find another way to do this?

Comment (webOS101) [inline src/Slider/Slider.js]: I'm open to suggestions.

**Comment by aarontam** [comment]
Closing in favor of https://github.com/enyojs/moonstone-extra/pull/97.

**Pull request closed without merging**